app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
remove ()

Reply:
done

Reply:
but after I removed the bracket, when I go to localhost:3000 it stated syntax error for that line

Reply:
oh solved already, I forgot the space